api/src/main/java/org/apache/iceberg/DataFile.java

```diff
@@ -139,8 +139,10 @@ static StructType getType(StructType partitionType) {
   DataFile copy();
 
   /**
-   * @return a list of offsets for file blocks if applicable, null otherwise. When available, this
-   * information is used for planning scan tasks whose boundaries are determined by these offsets.
+   * @return List of offsets for blocks of a file, if applicable, null otherwise.
+   * When available, this information is used for planning scan tasks whose boundaries
+   * are determined by these offsets.It is important that the returned list is sorted
```

Comment:
Nit: missing a space.

I think we should phrase the new content a little differently. "It is important" isn't very clear. I think it should be "offsets *must* be in sorted order."